### ISSUE TITLE
Whitescreen on startup

### ISSUE THREAD

**Issue by Lucas-Steinmann** [Zettlr/Zettlr#1350]
## Description
When starting Zettlr, only a white screen as in #826 and #846 is shown.
I could not start the console via ``Ctrl+Shift+I`` or ``F12`` or ``Ctrl+Alt+I``, so I hardcoded it into the js.
The stacktrace is:
```
Uncaught TypeError: Cannot destructure property 'Menu' of 'remote' as it is undefined.
    at Object.<anonymous> (/usr/lib/zettlr/renderer/zettlr-context.js:18)
    at Object.<anonymous> (/usr/lib/zettlr/renderer/zettlr-context.js:308)
    at Module._compile (internal/modules/cjs/loader.js:1145)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1166)
    at Module.load (internal/modules/cjs/loader.js:981)
    at Module._load (internal/modules/cjs/loader.js:881)
    at Function.Module._load (electron/js2c/asar.js:769)
    at Module.require (internal/modules/cjs/loader.js:1023)
    at require (internal/modules/cjs/helpers.js:77)
    at Object.<anonymous> (/usr/lib/zettlr/renderer/zettlr-body.js:24)
```

## Reproducing
Just start it. 
Issue just experienced since today.
Don't know what changed.

## Expected behaviour
UI is schown.

## Platform
 - OS and version: Arch Linux
 - Zettlr Version: 1.7.5-2 (from aur)

**Comment by Lucas-Steinmann** [comment]
I have no knowledge about javascript, so I don't know what it does there.
But it seems to try to access electron? 
I checked my package manager and the version of electron is 10.1.3-1.
I then checked my install log and saw that the version was bumped in an update today from 9 to 10.
Checked your issues on electron 10 and found this, so it is probably related: #858 

I uninstalled the electron package and installed electron9 and symlinked it to electron in ``/usr/bin`` and ``usr/lib``.
I installed it via aur and therefore had to change the dependency in the PKGBUILD.
It works again.

So this will probably affect all arch users as soon as they update.
